templates/patterns/list_item/list_item.ui_patterns.yml

```diff
@@ -84,7 +84,7 @@ list_item:
         end: 1546250400
         timezone: "Europe/Brussels"
     image:
-      type: "array"
+      type: "ImageValueObject"
```

Comment:
We cannot do this, we are not in charge of the API and we are breaking theirs. Let's just fix the tests.